### DISCUSSION TITLE
Unable to transfer domain out, invalid auth code.

### DISCUSSION THREAD

**Discussion by Froidland** [vercel/community#5795] [Account Issue]
### Question

I tried transferring my domain out from vercel to porkbun, but it gave me a bad authcode error.

![image](https://github.com/vercel/community/assets/7199366/3de8010f-cbf0-40f8-b585-a08e4d5e9557)
Is there any way to reset the auth code?

**Comment by amyegan** [comment]
Closing as a duplicate of https://github.com/vercel/community/discussions/5826
